lib/stdlib/src/timer.erl

```diff
@@ -399,37 +415,25 @@ handle_call({apply_once, {Started, Time, MFA}}, _From, Tab) ->
     {reply, Reply, Tab};
 %% Start an interval timer.
 handle_call({apply_interval, {Started, Time, Pid, MFA}}, _From, Tab) ->
-    NextTimeout = Started + Time,
-    TRef = monitor(process, Pid),
-    Reply = try
-                erlang:start_timer(
-                  NextTimeout,
-                  self(),
-                  {apply_interval, NextTimeout, Time, TRef, MFA},
-                  [{abs, true}]
-                 )
-            of
-                SRef ->
-                    ets:insert(Tab, {TRef, SRef}),
-                    {ok, {interval, TRef}}
-            catch
-                error:badarg ->
-                    demonitor(TRef, [flush]),
-                    {error, badarg}
-            end,
-    {reply, Reply, Tab};
+    {TRef, TPid, Tag} = start_interval_loop(Started, Time, Pid, MFA, false),
+    ets:insert(Tab, {TRef, TPid, Tag}),
+    {reply, {ok, {interval, TRef}}, Tab};
+handle_call({apply_repeatedly, {Started, Time, Pid, MFA}}, _From, Tab) ->
+    {TRef, TPid, Tag} = start_interval_loop(Started, Time, Pid, MFA, true),
+    ets:insert(Tab, {TRef, TPid, Tag}),
+    {reply, {ok, {interval, TRef}}, Tab};
 %% Cancel a one-shot timer.
 handle_call({cancel, {once, TRef}}, _From, Tab) ->
     _ = remove_timer(TRef, Tab),
     {reply, {ok, cancel}, Tab};
 %% Cancel an interval timer.
 handle_call({cancel, {interval, TRef}}, _From, Tab) ->
     _ = case remove_timer(TRef, Tab) of
-            true ->
-                demonitor(TRef, [flush]);
-            false ->
-                ok
-        end,
+        true ->
+            demonitor(TRef, [flush]);
+        false ->
+            ok
+    end,
```

Comment:
indentation seems spoiled?

Reply:
Hm... seems ok to me, what's wrong with it? :thinking::sweat_smile: (I'm using vim, not Emacs)

Reply:
In code you've changed you got patterns (`true ->`, `false ->`) intended in relation to `case`. it is different in you variant.
I don't think its related to Editor (although Emacs seems to do it the way it was by default).

proposal use if you like it
```
handle_call({cancel, {interval, TRef}}, _From, Tab) ->
    _ = case remove_timer(TRef, Tab) of
            true ->
                demonitor(TRef, [flush]);
            false ->
                ok
        end,
    {reply, {ok, cancel}, Tab};
```

Reply:
Ah, you mean to align on `case`, got it. I grepped through the stdlib, and in the modules there the alignment of the clauses and bodies is as you suggest. The alignment of the `end` differs sometimes, but is mostly as you suggest, too. So, yeah, I'll fix it here to fit in :wink:

Reply:
> or @juhlig

Not me :innocent::grin: